### DISCUSSION TITLE
navlist.group expandable, to be closed by default

### DISCUSSION THREAD

**Discussion by jlebherz** [livewire/flux#432] [Feature Requests]
we want to use navlist.group expandable
and the group should be closed by default, not expanded.
how can we achieve this?

    <flux:navlist.group heading="Account" expandable>
        <flux:navlist.item href="#">Profile</flux:navlist.item>
        <flux:navlist.item href="#">Settings</flux:navlist.item>
        <flux:navlist.item href="#">Billing</flux:navlist.item>
    </flux:navlist.group>

**Comment by tomhorvat** [comment]
You can use `expanded=""`
